packages/uploadthing/src/internal/handler.ts

```diff
@@ -223,135 +224,209 @@ export const buildRequestHandler = <
       return { status: 200 };
     }
 
-    if (!actionType || actionType !== "upload") {
+    if (!actionType || !VALID_ACTION_TYPES.includes(actionType)) {
       // This would either be someone spamming or the AWS webhook
       return new UploadThingError({
         code: "BAD_REQUEST",
         cause: `Invalid action type ${actionType}`,
-        message: `Expected "upload" but got "${actionType}"`,
+        message: `Expected ${VALID_ACTION_TYPES.map((x) => `"${x}"`)
+          .join(", ")
+          .replace(/,(?!.*,)/, " or")} but got "${"a"}"`,
```

Comment:
Damn that regex replace is hardcore :)

Reply:
Every other way I tried of doing it seemed like too much code hahah